### DISCUSSION TITLE
Maintenance Topic

### DISCUSSION THREAD

**Discussion by NuclearPhoenixx** [1coderookie/KobraGoNeoInsights#18] [Ideas]
There are multiple mentions of regular maintenance in the docs that one should of course do. However, there isn't a dedicated maintenance page (except for the Z axis maintenance). I guess there is too little "new" or unique things to the maintenance of the Go and Neo that would justify its own page, i.e. most generic 3D printer maintenance guides out there are usable, I guess. However, it would be nice to have just a small overview page maybe of some device-specific tipps and tricks as well as some links to guides. 🙂

**Comment by 1coderookie** [comment]
That's right, a dedicated maintenance page would be nice to have and it's actually on my todo list already, I just didn't find the time yet as I'd like to get the K2Pro site done first of all.   
Until then: maintenance tips can be found  
a) in the belonging sections in violet expandable textboxes in general and 
b) especially in the chapter Axes for the wheels as own menu items already - belt maintenance tips (as it's not a real 'maintenance' and more a check) are in an expandable textbox again.  
So I can probably just make an own menu item for those ones as well instead of having them in expandable textboxes which might get easily overseen as a quick workaround..

**Reply by 1coderookie** [reply]
Ok, so in the chapter Axes I rearranged some stuff and gave the maintenance parts an own menu item there.  
Besides that, I now made a first draft of a maintenance page, you can find it in the menu at the very top between Troubleshooting and Printserver or just click here: https://1coderookie.github.io/KobraGoNeoInsights/maintenance/  
It's not completely done yet, but the imho most important things are already being mentioned there now.

**Reply by NuclearPhoenixx** [reply]
Neat, thanks!

**Reply by 1coderookie** [reply]
Welcome :)  
Could you please mark the discussion as being answered then? Thx :)

**Reply by 1coderookie** [reply]
Thx, but I actually meant tagging/marking it as "answered", I think there must be a button for that as well somewhere..

**Reply by NuclearPhoenixx** [reply]
Nah there is only close, because it's technically not in the Q&A category. But it says "Closed as resolved", that should be good.

**Reply by 1coderookie** [reply]
Ah ok, well then it's fine :) 
Thx :)
